src/modules/export.js

```diff
@@ -256,6 +277,52 @@ export const closeExport = () => ({
   type: CLOSE
 });
 
+/**
+ * Update export fields
+ * @api public
```

Comment:
Did you find the `@api` js doc tags to be helpful?

Reply:
hah no, not really; I just noticed everything else in this package was using it and didn't want to break the tradition.

the reason why i don't find it helpful, i am not sure what `public` vs `private` means in a context of plugins like this. Is `private` only something that gets called within this file, or within the import-export plugin in general, so then `public` is just  `open-import` and `open-export`. If that makes sense \o/

Reply:
> private only something that gets called within this file

Yes :)